src/map/pc.hpp

```diff
@@ -730,6 +730,15 @@ struct map_session_data {
 	uint32* hatEffectIDs;
 	uint8 hatEffectCount;
 #endif
+
+	struct {
+		unsigned attack   : 1;
+		unsigned useitem  : 1;
+		unsigned chat     : 1;
+		unsigned sitstand : 1;
+		unsigned commands : 1;
+		unsigned npcclick : 1;
```

Comment:
Not a huge deal, but maybe make a note that `move` and `skill` restrictions are in `unit.hpp::unit_data`.

```suggestion
		// move and skill are defined in unit.hpp::unit_data as it can be applicable to all objects.
```